schemas/com.snowplowanalytics.snowplow.badrows/generic/jsonschema/1-0-0

```diff
@@ -0,0 +1,65 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Generic bad row",
+    "self": {
+      "vendor": "com.snowplowanalytics.snowplow.badrows",
+      "name": "generic",
```

Comment:
Don't have anything against just `generic`, but it feels that for consistency we can use at least a noun instead of adjective: generic error, generic failures or something of that kind?

Reply:
Hehe I also hesitated for this, I had first used `generic_error` and then switched to just `generic`. Switching back to `generic_error`